specification/1.0/README.md

```diff
@@ -120,7 +118,7 @@ Examples:
 }
 ```
 
-### Node referencing other nodes
+### Example: Node referencing other nodes
 
```

Comment:
And here as well :slightly_smiling_face:

Reply:
thumbs up! could you do it?